app/src/main/res/xml/accessibility_preferences.xml

```diff
@@ -24,4 +24,10 @@
         app:min="0"
         app:seekBarIncrement="1"
         app:showSeekBarValue="true" />
+    <SwitchPreference
```

Comment:
Can you also add `app:isVisible=false` so that non-nightly users don't see a jump of this dissappearing?